pkg/providers/v1/tags.go

```diff
@@ -310,3 +310,48 @@ func (t *awsTagging) buildTags(lifecycle ResourceLifecycle, additionalTags map[s
 func (t *awsTagging) clusterID() string {
 	return t.ClusterID
 }
+
+func (c *Cloud) TagResource(resourceId string, tags map[string]string) error {
+	request := &ec2.CreateTagsInput{
+		Resources: []*string{aws.String(resourceId)},
+		Tags:      buildAwsTags(tags),
+	}
+
+	_, err := c.ec2.CreateTags(request)
+
+	if err != nil {
+		klog.Errorf("Error occurred trying to tag resources, %v", err)
+		return err
+	}
+
+	return nil
+}
+
+func (c *Cloud) UntagResource(resourceId string, tags map[string]string) error {
```

Comment:
Same for this one.

Reply:
Same for https://github.com/kubernetes/cloud-provider-aws/pull/308#discussion_r833968442

Reply:
resolved offline